lib/utils.pm

```diff
@@ -1359,6 +1359,7 @@ sub reconnect_mgmt_console {
     my (%args) = @_;
     $args{timeout}             //= 300;
     $args{grub_expected_twice} //= 0;
+    permit_root_ssh();
```

Comment:
This can only be executed while being logged in as root on the machine via console to reconfigure ssh.

Reply:
Thank you for the hint! I thought that the used user is the root user (user with root access) because of the possibility to install.